spec/models/miq_policy_spec.rb

```diff
@@ -100,6 +100,16 @@
       end
     end
 
+    describe '#miq_event_definitions' do
+      before do
+        policy.miq_policy_contents.push(FactoryGirl.create(:miq_policy_content))
```

Comment:
This loops because `policy` is trying to be defined by `policy`.

If it's unrelated to the other specs, why not just add this to the accessor in the creation of `policy` above, around like 93?